awxkit/awxkit/api/pages/bulk.py

```diff
@@ -19,12 +19,3 @@ def post(self, payload={}):
 
 
 page.register_page(resources.bulk_job_launch, BulkJobLaunch)
-
-
-class BulkHostCreate(base.Base):
-    def post(self, payload={}):
-        result = self.connection.post(self.endpoint, payload)
-        return result.json()
```

Comment:
cli expects to return an awxkit Page object, not a dictionary

was resulting in `'list' object has no attribute '__dict__'` errors